pages/management/backup-index.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <landing-section-container>
+    <section>
+      <h2>
+        Management
+        <span class="text-primary">panel</span>
+      </h2>
+
+      <div class="flex flex-wrap justify-between py-2">
+        <div class="shadow rounded p-4">
+          <p class="text-primary small-text">Wallet Address</p>
```

Comment:
nitpick(non-blocking): i don't think that we need the p element, just add the class to the div